anesthetic/read/chain.py

```diff
@@ -31,6 +22,15 @@ def read_chains(root, *args, **kwargs):
     NestedSamples or MCMCSamples depending on auto-detection
 
     """
+    if 'burn_in' in kwargs:
+        raise KeyError(
+            "This is anesthetic 1.0 syntax. The `burn_in` keyword is no "
+            "longer supported in `read_chains`. You need to update, e.g.\n"
+            "read_chains(root, burn_in=0.5)         # anesthetic 1.0\n"
+            "read_chains(root).remove_burn_in(0.5)  # anesthetic 2.0\n"
+            "See also https://anesthetic.readthedocs.io/en/latest/"
+            "anesthetic.html#anesthetic.samples.MCMCSamples.remove_burn_in"
```

Comment:
We should make a note in #243 to update [this page](https://anesthetic.readthedocs.io/en/latest/reading_writing.html).

Reply:
Done in https://github.com/williamjameshandley/anesthetic/pull/243/commits/c69793e549e1db1c85c130e2a61bda495b70cdad